includes/import/abstract-wc-product-importer.php

```diff
@@ -294,7 +280,7 @@ protected function save_product_data( $product, $data ) {
 
 		// Purchase Note.
 		if ( isset( $data['purchase_note'] ) ) {
-			$product->set_purchase_note( wc_clean( $data['purchase_note'] ) );
+			$product->set_purchase_note( $data['purchase_note'] );
```

Comment:
Never mind. This gets run through `wc_clean` in the parser. I should have kept scrolling down. :P